File: 프로그래머스/1/17681. ［1차］ 비밀지도/［1차］ 비밀지도.java
class Solution {
    public String[] solution(int n, int[] arr1, int[] arr2) {
        String[] answer = new String[n];
        for(int i = 0; i < n; i++){
            int num = arr1[i] | arr2[i];
            answer[i] = "";
            for(int j = n - 1; j >= 0; j--){
                if((num & (1 << j)) != 0) answer[i] += "#";
                else answer[i] += " ";
            }
        }
        return answer;
    }
}

File: 프로그래머스/1/17681. ［1차］ 비밀지도/［1차］ 비밀지도.js
function solution(n, arr1, arr2) {
    let answer = new Array(n).fill("");
    for(let i = 0; i < n; i++){
        let num = arr1[i] | arr2[i];
        for(let j = n - 1; j >= 0; j--){
            if(num & (1 << j)) answer[i] += "#";
            else answer[i] += " ";
        }
    }
    return answer;
}